packages/terra-card/src/Card.scss

```diff
@@ -0,0 +1,27 @@
+
+/* stylelint-disable selector-class-pattern  */
+.card {
+  background-color: #fff;
+  border-radius: 3px;
+  box-shadow: 0 1px 1px 0 #1c1f21;
+  margin: 10px;
```

Comment:
Adding on to Jaime's comment above, I would reach out to @neilpfeiffer to see if we want a constant margin around each Card or if the Component the Card is within should control the spacing